hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -304,9 +326,23 @@ protected void deleteInstantFile(HoodieInstant instant) {
     }
   }
 
+  /**
+   * Many callers might not pass completionTime, here we have to search
+   * timeline to get completionTime
+   * TODO Depreciate this method and fix relate tests
+   */
+  private String getInstantFileName(HoodieInstant instant) {
```

Comment:
You mean the `instant` passed in is constructed manually instead of filtered from the timeline?